src/parse.js

```diff
@@ -276,6 +290,87 @@ export class CellParser extends Parser {
   }
 }
 
+export class TemplateCellParser extends CellParser {
+  constructor({tag, raw, ...options}, ...args) {
+    super(options, ...args);
+    this.tag = tag;
+    this.raw = raw;
+  }
```

Comment:
I think we don’t need these options: a TemplateCellParser should only be used to parse a template cell, and for parsing purposes we don’t actually care what the actual tag is, just that it’s a template cell. The tag and raw fields could be set by the caller on the returned node— they don’t need to be passed through as parser options if they don’t affect parsing.

```suggestion
```

Reply:
Makes sense!